src/controllers/reset.controller.js

```diff
@@ -0,0 +1,100 @@
+import { userService } from '../services/user.service.js';
+import { jwtService } from '../services/jwt.service.js';
+import { tokenService } from '../services/token.service.js';
+import { emailService } from '../services/email.services.js';
+import { Token } from '../models/token.js';
+import { Op } from 'sequelize';
+import bcrypt from 'bcrypt';
+import { User } from '../models/user.js';
+import { ApiError } from '../exceptions/api.error.js';
+import { validatePassword } from '../utils/methods.js';
+
+const requestReset = async (req, res) => {
+  const { email } = req.body;
+
+  if (!email) {
+    throw ApiError.notFound('Email is required');
```

Comment:
This should be a bad request error instead of not found, as the issue is with the request data, not the resource.